database/src/main/java/com/firebase/ui/database/OnChangedListener.java

```diff
@@ -0,0 +1,41 @@
+package com.firebase.ui.database;
+
+import com.google.firebase.database.ChildEventListener;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+
+public interface OnChangedListener {
```

Comment:
@puf, I would like that too, but you've chosen to keep `FirebaseArray` package private so people will get an error when overriding `onChanged` saying they can't access `FirebaseArray.OnChangedListener.EventType`. To put `OnChangedListener` in `FirebaseArray` we would have to go public and I don't think `FirebaseArray` is quite ready for that.